assets/css/_vehicle_card.scss

```diff
@@ -0,0 +1,52 @@
+.m-vehicle-card {
+  display: flex;
+  background-color: white;
+  border-radius: 1rem;
+  border: 1px solid black;
+  padding: 0.75rem;
+}
```

Comment:
**question:** this is probably as much of a design question as a dev question, but is there any overlap between this and our `<Card/>` component that was built for notificatiions but intended to be general-purpose?

Reply:
Yeah good question, I overlooked `<Card/>` in implementing. The only limitation right now with `Card` is that it has a required `Title` param which doesn't seem obvious to me looking at the low-fi designs but is maybe something that can be worked out with design as the fidelity increases.